### ISSUE TITLE
State list and vector drawables support

### ISSUE THREAD

**Issue by yaugenka** [syedowaisali/crystal-range-seekbar#154]
It would be nice to have a possibility to define a state list xml and vector drawables for thumbs.

**Comment by tejaskanani1** [comment]
Yes, it would be great to add custom drawables. We can easily modify drawables as per requirements.
